libbeat/docs/breaking.asciidoc

```diff
@@ -9,9 +9,152 @@ changes, but there are breaking changes between major versions (e.g. 5.x to
+
+// REVIEWERS: Would it better just to tell users to uses to change their
+// searches to use `beat.hostname` instead of `host`?
+
+If you have visualizations in Kibana that reference the `host` field, export the
+objects by using the Kibana API or UI, change any aggregation references to use
+`beat.hostname` instead of `host`, and then re-import the objects into Kibana.
+
+// REVIEWERS: Are the above details correct? Got them from a comment Pius made
+// on ruflin's doc.
+
+//TODO: Add links to the Kibana docs for exporting/importing objects.
+
+[[custom-template-non-versioned-indices]]
+===== Use case: You use a custom template and your indices are not versioned
+
+Mapping conflicts are likely in this use case because two different Beats
+versions (6.2 and 6.3) are sending data to the same index. For 6.2, Logstash
+adds the default `host` field, and for 6.3, Beats adds the `host.name` field,
+which results in a mapping conflict.
+
+*What do you need to change?*
+
+To resolve the mapping issue, do *one* of the following:
+
+* Use versioned indices to prevent the mapping conflict. In the Logstash
+pipeline configuration, set `manage_template => false` and use an index naming
+pattern that includes `[version]` metadata:
++
+[source,yaml]
+----
+manage_template => false
+index => "%{[@metadata][beat]}-%{[@metadata][version]}-%{+YYYY.MM.dd}"
+----
++
+For more information, see the documentation for the
+{logstash-ref}/plugins-inputs-beats.html[Beats input plugin].
+
+* Or, in the Beats config file, configure Beats to drop all `host.*` fields: 
++
+[source,yaml]
+----
+  processors:
+   - drop_fields:
+      fields: ["host"]
+----
++
+--
+IMPORTANT: If you drop the `host` fields, you cannot use the `add_host_metadata`
+processor.
+
+--
++
+With this configuration, Beats drops the `host` fields before sending the
+event to Logstash, and Logstash adds a default `host` field, as it did with
+previous Beats versions.
+
+[[beats-template-non-versioned-indices]]
+===== Use case: You use the Beats index template and your indices are not versioned
+
+In this use case, you load the Beats index template manually into Elasticsearch,
+and send your data through Logstash, but you don’t use the versioned index
+pattern to create versioned indices.
+
+You cannot resolve the problem by dropping the `host.*` fields, as described
+earlier, because Logstash will add a default `host` field, resulting in a
+mapping conflict because the index template defines the `host` field as an
+object. 
+
+*What do you need to change?*
+
+To resolve the mapping issue, do *one* of the following:
+
+// TOOD: Should add an example of the LS filter here, don't you think?
+
+* Drop the `host.*` fields in a Logstash filter. There will be no
+`host` field at ingestion time, and the mapping conflict is avoided.
+* Or:
+** Modify the Beats index template by removing the `host.*` fields, and
+** Configure Beats to drop all `host.*` fields, as described earlier.
++
```

Comment:
WRT "earlier".  Same concern applies (this document isn't designed to be read linearly). 

If we're referencing this: "In the Beats config file, configure Beats to drop all `host.*` fields: ", just say that instead of referencing it.

Reply:
I'll add a link to the section because I want users to know that the topic describes how to do this.